lib/openstudio-standards/standards/Standards.CoilHeatingDXSingleSpeed.rb

```diff
@@ -133,12 +133,23 @@ def coil_heating_dx_single_speed_standard_minimum_cop(coil_heating_dx_single_spe
       # TABLE 6.8.1D
       # COP = pthp_cop_coeff_1 - (pthp_cop_coeff_2 * Cap / 1000)
       # Note c: Cap means the rated cooling capacity of the product in Btu/h.
-      # If the unit's capacity is less than 7000 Btu/h, use 7000 Btu/h in the calculation.
-      # If the unit's capacity is greater than 15,000 Btu/h, use 15,000 Btu/h in the calculation.
-      capacity_btu_per_hr = 7000 if capacity_btu_per_hr < 7000
-      capacity_btu_per_hr = 15_000 if capacity_btu_per_hr > 15_000
+
+      # If the unit's capacity is nil or less than 7000 Btu/h, use 7000 Btu/h in the calculation
+      # If the unit's capacity is greater than 15,000 Btu/h, use 15,000 Btu/h in the calculation
+      if capacity_btu_per_hr.nil?
+        capacity_btu_per_hr = 7000.0
+        capacity_kbtu_per_hr = capacity_btu_per_hr / 1000.0
+        OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.CoilHeatingDXSingleSpeed', "Cooling Capacity for #{coil_heating_dx_single_speed.name}: #{sub_category} is nil. This zone may not have heating. Using default equipment efficiency for a 7 kBtu/hr unit.")
```

Comment:
Log messages should say 'heating capacity'

Reply:
Per 90.1 _"'Cap' means the rated cooling capacity of the product in Btu/h. If the unit’s capacity is less than 7000 Btu/h, use 7000 Btu/h in the calculation. If the unit’s capacity is greater than
15,000 Btu/h, use 15,000 Btu/h in the calculation."_
So cooling capacity is correct. I added an informative note to the message about it.